src/b3d/chisight/gen3d/inference/point_attribute_proposals.py

```diff
@@ -26,9 +29,20 @@ def propose_all_pointlevel_attributes(key, trace, inference_hyperparams):
     have shape (n_vertices,), log_q (a float) is (an estimate of)
     the overall log proposal density, and metadata is a dict.
     """
-    observed_rgbds_per_point = PixelsPointsAssociation.from_hyperparams_and_pose(
-        get_hypers(trace), get_new_state(trace)["pose"]
-    ).get_point_rgbds(get_observed_rgbd(trace))
+    if inference_hyperparams.in_inference_only_assoc_one_point_per_pixel:
```

Comment:
This gives a way to swap between whether to do pixel<>point association as before ("else" branch), or using the z buffering from the new unique image kernel.